chaos/cases/db.go

```diff
@@ -122,6 +122,8 @@ func ignoreExecSQLError(err error) bool {
 		return true
 	case errno.ErrDupEntry: // HACK: we tolerate `invalid connection`, then `Duplicate entry` may be reported.
 		return true
+	case errno.ErrTooBigRowsize: // HACK:we tolerate `Error 1118: Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535`
```

Comment:
```suggestion
	case errno.ErrTooBigRowsize: // HACK: we tolerate `Error 1118: Row size too large. The maximum row size for the used table type, not counting BLOBs, is 65535`
```

Reply:
is this a connection to MySQL? seems TiDB doesn't have such small length limit

https://docs.pingcap.com/zh/tidb/stable/tidb-limitations

Reply:
applied the suggestion in https://github.com/pingcap/dm/pull/1114/commits/91e5cf704830dbd3fc88d361a660f6b958f43b4d.

> is this a connection to MySQL?

YES

> seems TiDB doesn't have such small length limit

TiDB limited a row to 6MB (https://docs.pingcap.com/zh/tidb/stable/tidb-limitations#%E5%8D%95%E8%A1%8C%E7%9A%84%E9%99%90%E5%88%B6)